user-guide/install.md

```diff
@@ -1,37 +1,20 @@
 # Quick Start Installation Guide
 
-The Ambassador Edge Stack provides a comprehensive, self-service edge stack in the Kubernetes cluster with a decentralized deployment model and a declarative paradigm. So what does that mean, exactly?
-
-* **Comprehensive** allows you to take all the different pieces of your cluster, application, traffic, security, etc, and conveniently manage them all in one place, instead of having to individually configure them all, or their interactions with one another.
-* **Self-service** is the ability for an application developer to independently complete configurations without bothering their Operations team. For the Ambassador Edge Stack, this means app-devs can configure the traffic at the “edge” of their application.
-
-In other words, **the Ambassador Edge Stack is an all-in-one edge stack management tool** that allows developers and platform engineers alike to skip the red tape and get things done without stepping on each other’s toes.
-
-The Ambassador Edge Stack enables GitOps-style management of your application, in addition to the easy to use Edge Policy Console interface that visually displays all of your configurations.
-
-To start using the Ambassador Edge Stack and its features right away:
-
-1. [Install the Ambassador Edge Stack](#install-the-ambassador-edge-stack)
-2. [Add Hosts for Automatic HTTPS](#add-hosts-and-configure-tls)
-3. [Create and Verify Mappings](#create-mappings)
-4. [What's Next?](#whats-next)
+In this guide, we'll walk you through installing and configuring the Ambassador Edge Stack in your Kubernetes cluster. Within a few minutes, your cluster will be routing HTTPS requests from the Internet to a backend service. You'll also have a sense of how the Ambassador Edge Stack is managed.
 
 ## Before You Begin
 
 The Ambassador Edge Stack is designed to run in Kubernetes for production. The most essential requirements are:
 
-* Kubernetes Cluster 1.11 or later
-* `Kubectl`
-
-Find additional requirements and recommendations for a successful installation and deployment on the [Product Requirements](../product-requirements) page.
+* Kubernetes 1.11 or later
+* The `kubectl` command-line tool
 
 ## Install the Ambassador Edge Stack
 
-The Ambassador Edge Stack is deployed to Kubernetes from the command line via our custom manifest in a YAML file.
+The Ambassador Edge Stack is typically deployed to Kubernetes from the command line. If you don't have Kubernetes, you should use our [Docker](../../about/quickstart) to deploy the Ambassador Edge Stack locally. We also have [instructions for the Helm package manager](../helm).
```

Comment:
as an fyi, we have a warning with Helm that it is not supported with early access, so maybe want to put in a note here